autozone-poc/src/components/Blog/Blog.js

```diff
@@ -1,239 +1,87 @@
-
-import React, { Component, useState } from "react";
-import blog1 from "../../utlis/img/bg-img/blog1.jpg";
-import blog2 from "../../utlis/img/bg-img/blog2.jpg";
-import blog3 from "../../utlis/img/bg-img/blog3.jpg";
-import blog4 from "../../utlis/img/bg-img/blog4.jpg";
-import blog5 from "../../utlis/img/bg-img/blog5.jpg";
-import blog6 from "../../utlis/img/bg-img/blog6.jpg";
-import InnerBgImg from "../../utlis/img/bg-img/breadcumb2.jpg";
+import React, { useState } from "react";
 import "../Blog/Blog.css";
-import { NavLink } from "react-router-dom";
-
-const Blog = props => {
-  
-    return (
-    <div>
-      <div data-testid="background"
-        className="breadcumb_area breadcumb-style-two bg-img"
-        style={{ backgroundImage: "url(" + InnerBgImg + ")" }}
-      >
-        <div className="container h-100">
-          <div className="row h-100 align-items-center">
-            <div className="col-12">
-              <div className="page-title text-center">
-                <h2>Fashion Blog</h2>
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-      <div className="blog-wrapper section-padding-80">
-        <div className="container">
-          <div className="row">
-            <div className="col-12 col-lg-6">
-              <div className="single-blog-area mb-50">
-                <img src={blog1} alt="" />
-                <div className="post-title">
-                  <a href="#">
-                    Vivamus sed nunc in arcu cursus mollis quis et orci.
-                    Interdum et malesuada
-                  </a>
-                </div>
-                <div className="hover-content">
-                  <div className="hover-post-title">
-                    <a href="#">
-                      Vivamus sed nunc in arcu cursus mollis quis et orci.
-                      Interdum et malesuada
-                    </a>
-                  </div>
-                  <p>
-                    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-                    Fusce enim nulla, mollis eu metus in, sagittis fringilla
-                    tortor. Phasellus purus dignissim convallis.
-                  </p>
-                  <NavLink 
-                  to= {{
-                    pathname:"/singleblog",
-                    propsFor: {
-                      imgpath: blog1
-                    }
-                    }}>
-                    Continue reading <i className="fa fa-angle-right"></i>
-                    </NavLink>
-                </div>
-              </div>
-            </div>
+import { BrowserRouter as Router,NavLink } from 'react-router-dom';
+import list from "../Blog/blog.json";
```

Comment:
kindly use axios call or fetch call to implement to use the article in the page

Reply:
Added axios call